index.html

```diff
@@ -41,6 +41,111 @@ <h3 class="connect"><a href="#">LET'S CONNECT</a></h3>
 				</ul>
 			</div>
 		</section>
+		<section class="project-works">
+			<ul class="work">
+				<li>
+					<img src="Assets/images/Portfolio2.png" alt="tonic project">
+					<h2>Tonic</h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back-end dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
+			<ul class="big-project">
+				<li>
+					<img src="Assets/images/Portfolio4.png" alt="multipost-stories">
+					<h2>Multi-Post<span>Stories</span></h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back End Dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
+			<ul class="work">
+				<li>
+					<img src="Assets/images/Portfolio1.png" alt="Gymnastic">
+					<h2>Tonic</h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back-end dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
+			<ul class="big-project">
+				<li>
+					<img src="Assets/images/Portfolio3.png" alt="Professional-art">
+					<h2>Multi-Post<span>Stories</span></h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back End dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
+		</section>
+		<section class="contact">
+			<h1>About<span>Myself</span></h1>
+			<p>Hello I’m a software developer! I can help you build a product, feature or website Look through some of
+				my
+				work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+				contact
+				me.
+			</p>
```

Comment:
- [ ] There is a misalignment of the `p` tag section. Kindly refer to the attached screenshot and make the necessary changes. 

![Screenshot (7)](https://user-images.githubusercontent.com/64894927/120876321-d13ca900-c5b8-11eb-9c11-fdab527365bc.png)